proto/v1/iam_policy.proto

```diff
@@ -36,6 +37,8 @@ message IamPolicy {
   // If an etag is provided and does not match the current etag of the poliy,
   // the call will be blocked and an ABORTED error will be returned.
   string etag = 2;
+
+  ChangeLog change_log = 3;
```

Comment:
I think we can keep the change log in the audit service instead of the policy entity